### PR TITLE
Limelight with translation

### PR DESCRIPTION
**Pull request by TiaSinghania**
RPM problems with the shooter. This can be fixed with and RPM distance command or for our sake before the competition we can make small adjustments to the RPM.

### CODE REVIEW

**Comment by TiaSinghania** [comment]
Kanav-Aryan-Shivani

**Review by thetaspirit — APPROVED** [review]

File: src/main/java/frc/robot/commands/VisionAimingCommand.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  
+  /** Creates a new {@link VisionAimingCommand}. */
+  public VisionAimingCommand(SwerveDriveSubsystem subsystem, XboxController controller) {
+    super(subsystem, controller);
+  }
+
+  @Override
+  public void initialize() {
+    m_pid.reset();
+    // Limelight.setLed(3);
```
Comment (TiaSinghania) [inline src/main/java/frc/robot/commands/VisionAimingCommand.java]: Why isn't setLed ever called?

**Review by TiaSinghania — COMMENTED** [review]
I don't see setLed() being called anywhere in the code. Was this leftover from debugging?

File: src/main/java/frc/robot/commands/VisionAimingCommand.java (hunk 1 above)
Comment (thetaspirit) [reply src/main/java/frc/robot/commands/VisionAimingCommand.java]: > I don't see setLed() being called anywhere in the code. Was this leftover from debugging?

yeah, it's leftover from debugging.  should be added before merge

File: src/main/java/frc/robot/commands/VisionAimingCommand.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  @Override
+  public void initialize() {
+    m_pid.reset();
+    Limelight.setLed(3);
+    m_pid.setSetpoint(0.0); // 0.0 means the limelight is pointed at the right direction
+
+  }
+  //@Override
```
Comment (ethanlshen) [inline src/main/java/frc/robot/commands/VisionAimingCommand.java:28]: Why is this commented out?

File: src/main/java/frc/robot/commands/VisionAimingCommand.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.Limelight;
+import frc.robot.subsystems.SwerveDriveSubsystem;
+
+public class VisionAimingCommand extends SwerveJoystickCommand {
+  private final PIDController m_pid = new PIDController(0.03, 0, 0);
+  
+  /** Creates a new {@link VisionAimingCommand}. */
+  public VisionAimingCommand(SwerveDriveSubsystem subsystem, XboxController controller) {
+    super(subsystem, controller);
```
Comment (ethanlshen) [inline src/main/java/frc/robot/commands/VisionAimingCommand.java:18]: PID should be initialized here

**Review by ethanlshen — APPROVED** [review]

**Pull request merged**